modules/network/host/python/util.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Provides basic utilities for the network orchestrator."""
+import getpass
+import os
+import subprocess
+import shlex
+from common import logger
+import netifaces
+
+LOGGER = logger.get_logger('util')

```

Comment:
This seems to be the same util class used throughout the application. Can we copy that util class into this module before building?

Reply:
Removed for importing the common util class